.github/workflows/integrate.yml

```diff
@@ -179,6 +179,7 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       SLS_AWS_REQUEST_MAX_RETRIES: 21 #Increase number of retries, do to observed "Rate exceeded" fails
+      SERVERLESS_BINARY_PATH: ./dist/serverless-linux2
```

Comment:
Is the produced binary postfixed with `2`, do you know the reason?

Reply:
Good that you've caught it - I've added it so it's invalid to make sure that this env variable takes effect and forgot to revert :grimacing: